src/Microsoft.PowerShell.Commands.Utility/commands/utility/UnblockFile.cs

```diff
@@ -13,6 +13,7 @@
 
 #endregion
 
+#if !UNIX
```

Comment:
I'm fine with that but then we have effectively conditional compilation in both the source and the csproj files